ereolen_support/ereolen_support.form.inc

```diff
@@ -43,6 +43,10 @@ function ereolen_support_form($form, &$form_state) {
     [
       'type' => 'setting'
     ],
+    drupal_get_path('module', 'ereolen_support') . '/js/clearFields.js',
+    [
+      'type' => 'setting'
```

Comment:
The javascript here is not "settings" which is Drupal.settings in the frontend.

> Allowed values are 'file', 'inline', 'external' or 'setting'.

Think it should be file?

Reply:
Yeah, thats fixed